packages/babel-plugin-transform-object-rest-spread/src/index.js

```diff
@@ -39,7 +39,12 @@ export default function ({ types: t }) {
           args.unshift(t.objectExpression([]));
         }
 
-        path.replaceWith(t.callExpression(file.addHelper("extends"), args));
+        const helper = file.opts.polyfill === false ? {
+          type: "Identifier",
```

Comment:
Object.assign should probably be `MemberExpression` not just an object? you can use `t.memberExpression`